File: DigitHoles/solve/cpp/DigitHolesRunner.cpp
#include <topcoder>
using namespace TopCoder;
#include "DigitHoles.cpp"
#include <fstream>
using namespace std;

int main(int argc, char const *argv[]) {
    try {
        ifstream ifs(argv[1]);
        int number; read(ifs, number);
        ifs.close();

        ofstream ofs(argv[2]);
        show(ofs, numHoles(number));
        ofs.close();
    } catch (exception &e) {
        cerr << e.what() << endl;
    }
    return 0;
}

File: DigitHoles/solve/java/DigitHolesRunner.java
import java.io.FileReader;
import java.io.FileWriter;

import java.util.List;
import java.util.ArrayList;

import org.topcoder.TopcoderReader;
import org.topcoder.TopcoderWriter;
import org.topcoder.TypeRef;

public class DigitHolesRunner {
    public static void main(String[] args) {
    try {
        TopcoderReader reader = new TopcoderReader(new FileReader(args[0]));
        int number = (Integer) reader.next(Integer.class);
        reader.close();

        DigitHoles solver = new DigitHoles();
        TopcoderWriter writer = new TopcoderWriter(new FileWriter(args[1]));
        writer.write(solver.numHoles(number));
        writer.close();
    } catch (Exception err) {
        err.printStackTrace(System.err);
    }
    }    
}
